modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java

```diff
@@ -724,7 +724,9 @@ private Map<String, Object> _toProperties(
 					_dLFileEntryLocalService.fetchDLFileEntry(fileEntryId);
 
 				if (dlFileEntry != null) {
-					if (FeatureFlagManagerUtil.isEnabled("LPS-174455")) {
+					if (FeatureFlagManagerUtil.isEnabled(
+							objectDefinition.getCompanyId(), "LPS-174455")) {
+
 						fileEntry.setFileBase64(
```

Comment:
I think there is another `LPS-174455` in the Liferay code:

https://github.com/liferay/liferay-portal/blob/92b615c7ab129c57caed656b43254b3fcdc5ac89/modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java#L1355

could you check?